### PR TITLE
Adopters information in the readme

### PR DESCRIPTION
**Pull request by egekorkan**
Once we agree on this, I will replicate it in the other repositories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 [![codecov](https://codecov.io/gh/eclipse-thingweb/node-wot/branch/master/graph/badge.svg)](https://codecov.io/gh/eclipse-thingweb/node-wot)
 [![Telegram Group](https://img.shields.io/endpoint?color=neon&url=https%3A%2F%2Ftg.sumanjay.workers.dev%2Fnodewot)](https://t.me/nodewot)
 [![Discord](https://img.shields.io/badge/Discord-7289DA?logo=discord&logoColor=white&label=node-wot)](https://discord.gg/JXY2Jzefz3)
+![Static Badge](https://img.shields.io/badge/Show%20Adoption%20and%20Usage-%2331b8a3ff?logoColor=31b8a3ff&link=https%3A%2F%2Fgithub.com%2Feclipse-thingweb%2Fnode-wot%23adopters)
```
Comment (relu91) [inline README.md:16]: is it me or this badge does not link where it should suppose to link?

Comment (egekorkan) [reply README.md]: shields.io generated link was wrong somehow... Fixed :)

**Review by relu91 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -13,6 +13,7 @@
 [![codecov](https://codecov.io/gh/eclipse-thingweb/node-wot/branch/master/graph/badge.svg)](https://codecov.io/gh/eclipse-thingweb/node-wot)
 [![Telegram Group](https://img.shields.io/endpoint?color=neon&url=https%3A%2F%2Ftg.sumanjay.workers.dev%2Fnodewot)](https://t.me/nodewot)
 [![Discord](https://img.shields.io/badge/Discord-7289DA?logo=discord&logoColor=white&label=node-wot)](https://discord.gg/JXY2Jzefz3)
+[![Static Badge](https://img.shields.io/badge/Show%20Adoption%20and%20Usage-%2331b8a3ff?logoColor=31b8a3ff)](https://github.com/eclipse-thingweb/node-wot#adopters)
```
Comment (JKRhb) [inline README.md:16]: The changes look good to me :) How about something "Adaptors and users" for the label, though? The the leading "Show..." is a bit of a deviation from the other badges.

Comment (egekorkan) [reply README.md:16]: That is much nicer indeed!

File: README.md [hunk 3]
```diff
@@ -432,6 +433,12 @@ sudo ln -sf /usr/local/n/versions/node/<VERSION>/bin/node /usr/bin/node
 
 Please check out our [contributing guidelines](CONTRIBUTING.md) for more details.
 
+## Adopters
+
+If you are using Eclipse Thingweb node-wot within your organization, please support us by adding your logo to the [Eclipse IoT adopters list](https://iot.eclipse.org/adopters/).
```
Comment (danielpeintner) [inline README.md:438]:
```suggestion
If you are using Eclipse Thingweb node-wot within your organization, please support us by adding your logo to the [Eclipse IoT adopters list](https://iot.eclipse.org/adopters/#iot.thingweb).
```

Does it make sense to point to Thingweb directly

File: README.md [hunk 4]
```diff
@@ -432,6 +433,12 @@ sudo ln -sf /usr/local/n/versions/node/<VERSION>/bin/node /usr/bin/node
...
+To do so, simply open an issue at [the Eclipse Gitlab](https://gitlab.eclipse.org/eclipsefdn/it/api/eclipsefdn-project-adopters/-/issues/new?issuable_template=adopter_request) by providing your the name of the organization, its logo, and a link to your organization or team.
```
Comment (danielpeintner) [inline README.md:439]:
```suggestion
To do so, simply open an issue at [the Eclipse Gitlab](https://gitlab.eclipse.org/eclipsefdn/it/api/eclipsefdn-project-adopters/-/issues/new?issuable_template=adopter_request) by providing the name of your organization, its logo, and a link to your organization or team.
```

Sentence "as is" reads weird ...

**Review by danielpeintner — CHANGES_REQUESTED** [review]
Some comments below

File: README.md (hunk 4 above)
Comment (egekorkan) [reply README.md:439]: must be some leftovers on my plate :)

**Review by danielpeintner — APPROVED** [review]
LGTM now.

**Pull request merged**